assets/js/theme/cart/shipping-estimator.js

```diff
@@ -17,12 +17,19 @@ export default class ShippingEstimator {
     }
 
     initFormValidation() {
+        const shippingEstimatorAlert = $('.shipping-quotes');
+
         this.shippingEstimator = 'form[data-shipping-estimator]';
         this.shippingValidator = nod({
             submit: `${this.shippingEstimator} .shipping-estimate-submit`,
         });
 
         $('.shipping-estimate-submit', this.$element).on('click', event => {
+            if (shippingEstimatorAlert.length && shippingEstimatorAlert.attr('role')) {
+                shippingEstimatorAlert.removeAttr('role');
+            }
```

Comment:
can we avoid double check here? 
Because we are using jQuery and shippingEstimatorAlert its node, we can just check hippingEstimatorAlert.attr('role'). If this attribute will not be present, undefined will be returned value and condition will work

Reply:
agree. good point